blazor/datagrid/column-menu.md

```diff
@@ -70,6 +70,65 @@ The default menu items are displayed in the following table,
 The following image represents DataGrid with column menu property enabled,
 ![Blazor DataGrid with Column Menu](images/blazor-datagrid-column-menu.gif)
 
+## Custom column menu
+
+Custom column menu items can be added by defining [ColumnMenuItems](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_ColumnMenuItems) as a collection of [ColumnMenuItemModel](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ColumnMenuItemModel.html) class. Actions for the customized column menu item can be defined in the [ColumnMenuItemClicked](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_ColumnMenuItemClicked) event of the Grid.
+
+In the following sample, sorting will be cleared when clicking the custom column menu item using the [ClearSortingAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_ClearSortingAsync) method in the `ColumnMenuItemClicked` event.
```

Comment:
concept is good. enable multiple initial sorting and clear that specific column sorting alone. https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_ClearSortingAsync_System_Collections_Generic_List_System_String__